chart/flux/templates/deployment.yaml

```diff
@@ -226,7 +226,10 @@ spec:
           - --git-verify-signatures={{ .Values.git.verifySignatures }}
           - --git-set-author={{ .Values.git.setAuthor }}
           {{- if .Values.git.secret.enabled }}
```

Comment:
this line should be deleted, as it causes a syntax error:
two opening if's but only one end.
This line is made obsolete by the next line :)